apps/sseramemes/.gitignore

```diff
@@ -0,0 +1 @@
+test.png
```

Comment:
does this .gitignore works?

Reply:
Yes, it works.

![image](https://user-images.githubusercontent.com/16626980/170763701-64e6e5d3-c7bf-4a6f-87d1-4abbdda5949e.png)